Libraries/DeviceInfo/DeviceInfo.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DeviceInfo
+ */
+'use strict';
+
+var RCTDeviceInfo = require('NativeModules').DeviceInfo;
+
+/**
+ * `DeviceInfo` gives you information about your running application and the
+ * device you are running on.
+ *
+ * #### Device Name on Android
+ *
+ * If you want to get the device name on Android then you will need to add the
+ * bluetooth permission to your application's `AndroidManifest.xml` file.
+ *
+ * ```
+ * <uses-permission android:name="android.permission.BLUETOOTH"/>
+ * ```
+ */
+var DeviceInfo = {
+  /**
+   *  Return a unique ID for the current device (this is the IDFV on iOS so it will change if all apps from the current apps vendor have been previously uninstalled)
+   */
+  get UniqueID() { return RCTDeviceInfo.uniqueId; },
```

Comment:
Why do these need to be getters? Flow doesn't support `get` and `set`. If we just use them directly, we can annotate them with Flow.